src/main/java/org/vanilladb/core/storage/file/FileMgr.java

```diff
@@ -195,16 +197,12 @@ void write(BlockId blk, IoBuffer buffer) {
 	BlockId append(String fileName, IoBuffer buffer) {
 		try {
 			IoChannel fileChannel = getFileChannel(fileName);
-
-			// Rewind the buffer for writing
-			buffer.rewind();
-
-			// Append the block to the file
-			long newSize = fileChannel.append(buffer);
-
+			
+			// Optimization: Doesn't actually append to file
+			long pageId = fileChannel.append(buffer);

```

Comment:
This should be called `blockId` since `Page` stands for a memory slot that holds a file block.